slides/index.html

```diff
@@ -32,49 +44,108 @@
 # Dette kurset
 
 ## 3 timer
+
+Blåtann  
+mbed  
+Hjelpemidler
+
 ## 3 oppgaver
 
+Lag en mbed med blåtann  
+Chat fra telefonen til mbed-en  
+Something, something, third task
+
 ???
 
-Vi tar en pause mellom hver oppgave
+- Vi holder korte foredrag om teknologi fulgt av oppgaver
+- Under oppgavene går vi rundt og hjelper til, rekk opp hånda om dere trenger hjelp
+- Vi tar ca et kvarters pause mellom hver oppgave
 
 ---
 class: center
 
 # http://bftj.co
 
-*screenshot*
+![](/images/bftjco.png)
 
 ???
 
-Vi har en nettside, som inneholder guider og oppgavene etc.
+- Vi har en nettside, som inneholder **guider**, **oppgavene**, **slides** osv
+- Bruk denne som ressurs når du jobber
+- Denne er tilgjengelig også etter kurset så den kan brukes om du trenger hjelp senere
 
 ---
-class: center
+class: center, middle
 
-# Bluetooth
+# Blåtann
 
-trådløs teknologi, blabla
+Trådløs teknologi  
+
+Kort rekkevidde  
+
+Sender data mellom enheter
 
 ???
 
-Få med det mest grunnleggende
+- Bluetooth er en kommunikasjonsprotokoll for å kommunisere mellom enheter
+- Kort rekkevidde på kommunikasjonen
+- Starter med å annonsere at den finnes -> med info som navn, adresse og liknende
+- Når noen kopler seg til forteller den hva den kan tilby (hvilke services den har)
 
 ---
-class: center
+class: center, middle
 
 # mbed
 
+Lavterskeltilbud til lavlevelprogrammering  
```

Comment:
'lavlevel' is an awkward combo of norwegian and english. either 'lavnivåprogrammering' or 'low level programming'